crates/parser/src/abi.rs

```diff
@@ -568,6 +568,21 @@ def_instruction! {
             ty: TypeId,
         } : [1] => [1],
 
+        /// TODO
```

Comment:
TODO comment here and below.

Reply:
Gah sorry I swear I'm not forgetting these intentionally, despite this being present in every PR I've sent so far...